.github/build.yaml.gomplate

```diff
@@ -277,6 +276,7 @@
         run: |
           export DOCKER_INSTALL=true
           sudo -E make add_local_repo
+      {{{ tmpl.Exec "make" "deps_ci" }}}
```

Comment:
we have been into this already several times... hoping it's the good one :crossed_fingers:

Reply:
No this not the good one... 😞  this will break arm64 builds, we are still trapped on #1515, I just added it to see if I could manage to get a green CI for x86_64 at least. But the CI won't be green due to https://github.com/rancher/elemental-toolkit/pull/1518#issuecomment-1157652688

Pretty disappointed, we have several things to digest out of this issue chain of issues.

Reply:
correct, indeed I'd treat https://github.com/rancher/elemental-toolkit/issues/1515 separately and forget about it at the moment. We weren't capable of doing that before, so even if frustrating, I'd say to just let it go..

Reply:
yes gonna remove this commit, just I considered it when I thought it was enough to get a green CI...